projects/core/build/tokens.ts

```diff
@@ -521,11 +521,11 @@ const aliases = {
         highlight: token(color.blue[700]),
       },
       color: {
-        value: token(color.construction[700]),
+        value: token(color.construction[500]),
```

Comment:
`--cds-alias-object-interaction-color`

Reply:
I'm not familiar with this change. Need some context. Possibly requested by another designer.

Reply:
This looks like where we decided to make the interactive icons dark grey. Blue is only for when an item like an icon actually takes you to a different page, and is acting like a link.

Reply:
Can we confirm this is the case?

Reply:
This value is from the alias mapping spreadsheet. The colors used for interactive icons aren't reflected here, as that's a component-level change. This change is a system-level change where the `--cds-alias-object-interaction-color` value used to be mapped to construction 700, but the spreadsheet indicates it should be mapped to construction 500 now.

Reply:
I checked this. This is what it's mapped to in our light theme Figma file and is correct with the spreadsheet. 
<img width="559" alt="image" src="https://github.com/vmware-clarity/core/assets/930622/37df394f-cea2-4cbc-9455-e79c9a1de44e">

Reply:
@Partick @colinreedmiller Can you take a look at this change?

Reply:
First off, I am sorry for the headache that this is causing. 

It appears that `-cds-alias-object-interaction-color` is being applied to text. The change to the mapping is making the text lighter and breaking contrast.  `--cds-alias-object-interaction-color` is only contextually appropriate for non-text items. We're mainly specifying it for graphical items like icons. 

We can fix this by specifying `--cds-alias-typography-color-300` (which maps to the global Construction 800 in light theme) for the default state and ` --cds-alias-typography-color-500` for the selected state. 

Typography Color 300 meets contrast.

<img width="472" alt="image" src="https://github.com/vmware-clarity/core/assets/930622/aeb95bc8-0011-46d6-8e87-5d4a372fd1e4">

Reply:
So what's happened is that we've moved away from the stated pattern of using interaction colors for text. It made more contextual sense to use typography colors. However, this allowed for the breakage we're experiencing now.

Reply:
I'm reading this thread to understand the new rules, in the clarity core storybook page that @astorije linked the tab example has text next to an icon, are you saying the icon should be a different color from the text?
![Screen Shot 2023-07-25 at 5 35 03 PM](https://github.com/vmware-clarity/core/assets/1950438/944e11c6-496b-4e6b-8e6e-e8ebb8bdaa2c)

There are also some existing components using `--cds-alias-object-interaction-color` for text color, I haven't done an exhaustive check but two examples are `cds-navigation-item` and `cds-accordion-header`. Are these going to be updated with a new color?

Perhaps it would make sense to update `--cds-alias-object-interaction-color` to match the typography color and provide a new variable for the non-text use case? As a bonus, this would significantly ease the breaking change and it would give time to document when the new color should be used

Reply:
@dturcotte @astorije We're reverting this change https://github.com/vmware-clarity/core/pull/255. Thanks for the feedback

Reply:
sounds good, thank you!